include/eigenpy/sparse/eigen-from-python.hpp

```diff
@@ -156,6 +156,10 @@ void eigen_sparse_matrix_from_py_construct(
     }
     MapMatOrRefType sparse_map(m, n, nnz, indptr.data(), indices_ptr, data_ptr);
 
+#if EIGEN_VERSION_AT_LEAST(3, 4, 90)
+    sparse_map.sortInnerIndices();
```

Comment:
Probably at some point we would like to add a check (and throw an exception) if the indices are unsorted and the Eigen version is below 3.4.90 ?

Reply:
Yes, sure.